doc/help/MOUNT.md

```diff
@@ -0,0 +1,41 @@
+kdb-mount(1) - Mount a file to the key database
+===============================================
+
+## DESCRIPTION
+
+This command allows a user to mount a new "backend".  
```

Comment:
And please write about that kdb-mount writes into /etc/ and you need to be root to use kdb-mount.